lib/actions/purgeTable.js

```diff
@@ -0,0 +1,77 @@
+let tableName = null;
+let dynamodb = null;
+
+const findPrimaryKey = () => {
+  const params = {
+    TableName: this.tableName
+  }
+
+  return new Promise((resolve, reject) => {
```

Comment:
No need for this extra promise. Just return object from body of `then`.

Reply:
Not sure why you've closed this issue before.
What I meant is to do this simpler code instead:

```
return context.dynamodb.describeTable(params).promise()
      .then(tableDescription => {
        context.primaryKey = tableDescription.Table.KeySchema.filter((element) => element.KeyType === "HASH")[0].AttributeName
        return context
      })
```